app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    parts = command.split()
+
+    if len(parts) != 3:
+        return
+
+    mv, filename, destination_path = parts
+    path, new_filename = os.path.split(destination_path)
+
+    if mv != "mv":
+        return
+
+    if not os.path.exists(filename):
+        return
+
+    if destination_path.endswith("/"):
```

Comment:
omit checking explicitly for "/". What if you have different OS?

Reply:
sorry, but i can`t understand how to do it, can you give me a hint?